robot/impl/resource_manager.go

```diff
@@ -716,26 +724,30 @@ func (manager *resourceManager) updateResources(
 	if err := cleanAppImageEnv(); err != nil {
 		manager.logger.Errorw("error cleaning up app image environement", "error", err)
 	}
-	for _, p := range config.Added.Processes {
-		_, err := manager.processManager.AddProcessFromConfig(ctx, p)
-		if err != nil {
-			manager.logger.Errorw("error while adding process, skipping", "process", p.ID, "error", err)
-			continue
-		}
-	}
-	for _, p := range config.Modified.Processes {
-		if oldProc, ok := manager.processManager.RemoveProcessByID(p.ID); ok {
-			if err := oldProc.Stop(); err != nil {
-				manager.logger.Errorw("couldn't stop process", "process", p.ID, "error", err)
+	if !manager.opts.disableProcesses {
```

Comment:
can we just error out fast if someone does this? it seems like more a violation than skipping things